ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessActivityResource.scala

```diff
@@ -66,29 +63,29 @@ class AttachmentResources(attachmentService: ProcessAttachmentService,
     path("processes" / Segment / VersionIdSegment / "activity" / "attachments") { (processName, versionId) =>
       (post & processId(processName)) { processId =>
         canWrite(processId) {
-          fileUpload("attachment") { case (metadata, byteSource) =>
-            complete {
-              attachmentService.saveAttachment(processId.id, versionId, metadata.fileName, byteSource)
+          withoutSizeLimit { // we have separate size limit validation inside attachmentService
```

Comment:
Why don't we use https://doc.akka.io/docs/akka-http/current/routing-dsl/directives/misc-directives/withSizeLimit.html?

Reply:
because it returns 500 with ugly message
```
EntityStreamSizeException: incoming entity size (8842875) exceeded size limit (1 bytes)! This may have been a parser limit (set via `akka.http.[server|client].parsing.max-content-length`), a decoder limit (set via `akka.http.routing.decode-max-size`), or a custom limit set with `withSizeLimit`.
```

Reply:
ok, I see. Are we sure we process it in safe-enough way so that we won't blow designer if sb uploads 2GB file?

Reply:
I hope so. We use akka implemenation of InputStream `akka.stream.impl.io.InputStreamSinkStage` underneath, which handles correctly stream termination